ts/src/binance.ts

```diff
@@ -9040,6 +8947,12 @@ export default class binance extends Exchange {
                 impliedNetwork = this.safeString (conversion, impliedNetwork, impliedNetwork);
             }
         }
+        // actually, we do not need the manually defined implied networks and deriving the networks by them
+        // we can just dynamically find them
+        if (impliedNetwork === undefined) {
+            // so, if value was not changed from hardcoded value
+            impliedNetwork = this.getNetworkCodeByNetworkUrl (code, url);
+        }
```

Comment:
across all coins, inside their "network" entries, each of them have some 'base-url' so we can dynamically use them to match the networks (so this approach is superset compared to the "reversedNetworks" hardcoded dictionary under 'options')